lib/js/components/RichList/RichListItem/RichListItem.vue

```diff
@@ -235,14 +270,45 @@ $rich-list-item-icon-colors: (
 		}
 	}
 
-	&__trailingSlot {
-		align-items: center;
+	&__rightContainer {
+		align-items: flex-start;
```

Comment:
It looks off, especially on mobile, when this container has property align-items of value flex-start and content container has center. Right icons and left icon look misaligned as well.

Reply:
But this is correct, this is how Ewa wanted it for the mobile view.

Reply:
yes, because in figma there is different content in Content and ACS - that's why it is displayed differently there

Reply:
![Zrzut ekranu 2024-03-19 o 14 21 09](https://github.com/bethinkpl/design-system/assets/4587534/d1b32840-5386-4c36-a94e-79dcb1803813)
![Zrzut ekranu 2024-03-19 o 14 21 00](https://github.com/bethinkpl/design-system/assets/4587534/e6899e18-3a55-4616-a65a-594caf215d33)